adtl/transformations.py

```diff
@@ -0,0 +1,19 @@
+""" Functions which can be applied to source fields, allowing extensibility """
+
+from datetime import datetime
+import pint
+
+
+def isNotNull(value):
+    True if value != None and value != "" else False
```

Comment:
needs a return, simpler to use `return value not in [None, ""]`, boolean expressions do not need True / False annotations.